### PR TITLE
BOAC-1509: adds filtered cohorts to home page

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-1509

### CODE REVIEW

File: src/components/search/SortableStudents.vue [hunk 1]
```diff
@@ -138,8 +138,7 @@
             <span class="sr-only">{{ srText['majors[0]'] }}</span>
             <div v-if="student.majors.length === 0">--<span class="sr-only">No data</span></div>
             <div v-for="major in student.majors"
-                 v-bind:key="major"
-                 v-if="student.majors.length > 0">{{ major }}</div>
+                 v-bind:key="major">{{ major }}</div>
```
Comment (johncrossman) [inline src/components/search/SortableStudents.vue]: This `v-bind:key` can be `:key`

File: src/views/Home.vue [hunk 2]
```diff
@@ -1,11 +1,50 @@
...
-    Welcome home!
+  <div class="home-container">
+    <Spinner/>
+    <div class="home-content" v-if="!loading">
+      <div>
+        <div id="filtered-cohorts-header-row" class="home-page-section-header-wrapper">
+          <h1 class="page-section-header" v-if="myCohorts && !myCohorts.length">
+            You have no saved cohorts.
+          </h1>
+          <h1 class="page-section-header" v-if="myCohorts && myCohorts.length">
```
Comment (johncrossman) [inline src/views/Home.vue]: @lyttam Our `Util` mixin aliases lodash `_.size`. You could use it (twice) above: `v-if="size(myCohorts)"`

File: src/views/Home.vue [hunk 3]
```diff
@@ -1,11 +1,50 @@
...
+            Cohorts
+          </h1>
+        </div>
+        <div v-if="myCohorts && !myCohorts.length">
+          <a id="create-filtered-cohort" href="/cohort/filtered">Create a student cohort</a>
```
Comment (johncrossman) [inline src/views/Home.vue]: Ditto on using `router-link` and `/cohort/${...}`

**Comment by johncrossman** [comment]
Welcome home `HomeCohort`! Looks good – thanks @lyttam. I put a few suggestions.

**Comment by lyttam** [comment]
Thanks @johncrossman. Amended.

**Pull request merged**